docs/user-guide/transformations/joins.md

```diff
@@ -218,11 +218,7 @@ If you want to make sure that only quotes within a certain time range are joined
 the `tolerance` argument. In this case we want to make sure that the last preceding quote is within 1 minute of the
 trade so we set `tolerance = "1m"`.
 
-=== ":fontawesome-brands-python: Python"
```

Comment:
I don't understand. Why is this removed?

Reply:
Python and Rust logos are already part of the auto-generated `code_block` template. No need for another Python logo.

Reply:
Alright, thanks!